src/main/java/com/fasterxml/jackson/databind/deser/BasicDeserializerFactory.java

```diff
@@ -1370,9 +1371,12 @@ protected CollectionType _mapAbstractCollectionType(JavaType type, Deserializati
                     MapDeserializer md = new MapDeserializer(type, inst, keyDes, contentDeser, contentTypeDeser);
                     JsonIgnoreProperties.Value ignorals = config.getDefaultPropertyIgnorals(Map.class,
                             beanDesc.getClassInfo());
-                    Set<String> ignored = (ignorals == null) ? null
-                            : ignorals.findIgnoredForDeserialization();
+                    Set<String> ignored = (ignorals == null) ? null : ignorals.findIgnoredForDeserialization();
```

Comment:
If possible, please do not change indentation and other aspects as this adds to diff size.